src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -137,6 +137,9 @@ private async Task<(IWebDriver browser, ILogs log)> CreateBrowserAsync(string co
         {
             var opts = new ChromeOptions();
 
+            // Force language to english for tests
+            opts.AddUserProfilePreference("intl.accept_languages", "en");
```

Comment:
At least for the BasicTestApp, we force the app to be en-us by default. For server app, the browser language might not make a difference (I don't think we register the localization middleware). Should we modify the apps to configure a default culture instead?

Reply:
Sure, 

I notice we do have some localization in https://github.com/dotnet/aspnetcore/blob/haok/blazor/src/Components/test/testassets/TestServer/InternationalizationStartup.cs

Should I go ahead and add it to all the various startups in TestServer only, or should I add it to all Startups under test assets?

Also so we don't want to also set it in the ChromeOptions for redundancy?